napari/_qt/qt_dims.py

```diff
@@ -328,3 +329,128 @@ def focus_down(self):
         else:
             index = (displayed.index(self.last_used) - 1) % len(displayed)
             self.last_used = displayed[index]
+
+    def play_dim(self, axis: int = 0, fps: float = 10, range=None):
```

Comment:
I would rather this be named `play` - its already located on the `dims` view, and matches nicer with `stop`

Reply:
changed